### ISSUE TITLE
Convert IRCamera input to triplet (x,y,int)

### ISSUE THREAD

**Issue by ImJoaoPedro** [ImJoaoPedro/RaspArd-FireFighter#7]

**Comment by jtaca** [comment]
The goal is to output a triplet to the pi?

**Comment by ImJoaoPedro** [comment]
Yes, só calculations can be made there and sent to RoMeo / Cherokey

**Comment by jtaca** [comment]
why the RoMeo? why a triplet, why not a tuple? why not only output if in fact a heat source is found? Avoiding spam and freeing up the connection for other data.
